docs/user/04-metrics.md

```diff
@@ -1,3 +1,395 @@
 # Metrics
 
-The feature is not available yet. To understand the current progress, watch this [epic](https://github.com/kyma-project/kyma/issues/13079).
+>**Note**: The feature is not available yet. To understand the current progress, watch this [epic](https://github.com/kyma-project/kyma/issues/13079).
+
+
+Observability is all about exposing the internals of components in a distributed application and making that internal analyzable at a central place.
+While application logs and traces are usually request oriented, metrics are aggregated statistics exposed by a component to reflect the internal state. Typical statistics like the amount of processed requests, or the amount registered users can be very useful to introspect the current state and also the health of a component. Also, it allows to define proactive and reactive alerts in case of metrics are about to reach thresholds soon or in case they already passed thresholds.
+
+The goal of the Telemetry Module is to support you in collecting all relevant metrics of workload in a Kyma cluster and ship them to a backend for further analysis. Hereby, relevant Kyma modules like Istio or Serverless will contribute instantly and typical enrichment of the data is happening. Multiple [vendors for OTLP-based backends](https://opentelemetry.io/ecosystem/vendors/) are available.
+
+## Prerequisites
+
+A component from which you want to collect metrics data, needs to expose (or instrument) the metrics first. Typically, it instruments specific metrics for the used language runtime (like nodeJS) and custom metrics specific to the business logic. Also, the exposure can be in different formats, usually the pull-based prometheus format or the [push-based OTLP format](https://opentelemetry.io/docs/specs/otlp/).
+
+To do the instrumentation you usually leverage an SDK, namely the [prometheus-client libraries](https://prometheus.io/docs/instrumenting/clientlibs/) or the [Open Telemetry SDKs](https://opentelemetry.io/docs/instrumentation/). Both libraries provide extensions to activate language specific auto-instrumentation like for nodeJS and an API to implement custom instrumentation.
+
+## Architecture
+
+The Telemetry module provides an in-cluster central Deployment of an [OTel Collector](https://opentelemetry.io/docs/collector/) acting as a gateway. The gateway exposes endpoints for the [OTLP protocol](https://opentelemetry.io/docs/specs/otlp/) for GRPC and HTTP-based communication using the dedicated `telemetry-otlp-metrics` service, where all Kyma components and users' applications should send the metrics data to.
+
+Optional, it provides a DaemonSet of an [OTel Collector](https://opentelemetry.io/docs/collector/) acting as an agent. That agent will scrape metrics of workload in the [prometheus pull-based format](https://prometheus.io/docs/instrumenting/exposition_formats) and can provide runtime specific metrics for workload.
+
+![Architecture](./assets/metrics-arch.drawio.svg)
```

Comment:
In the diagram, some connector descriptions have white background. To match the other ones, they need transparent background.
Also, "Prometheus" must be capitalized

Reply:
fixed